docs/guides/cloud/debugging/test-replay.mdx

```diff
@@ -0,0 +1,136 @@
+
+- What Test Replay is and why you should use it
+- How to get started with Test Replay in Cypress Cloud
+- How to debug failed and flaky CI test runs more accurately and efficiently
+
+:::
+
+## What Is Test Replay?
+
+Test Replay in Cypress Cloud allows developers to accurately and efficiently debug failed and flaky CI test runs. It captures every detail of test runs, enabling developers to replay and interact with the DOM, inspect network events, and console events as they happened during CI. This eliminates the need to replicate failed tests locally and enhances the ability to debug complex applications with confidence and speed.
+
+## Why Test Replay?
+
+Before Cypress `v13`, test failures in CI have historically been captured through screenshots, videos, and stack trace outputs, but these artifacts provide limited information. Developers often need more context to understand the root cause of failures, requiring them to reproduce the issue locally, which may not always be possible due to environmental or network conditions. This knowledge gap between test failures and debugging capabilities leads to increased debugging time and potentially deploying unresolved bugs to production.
+
+### How Does Test Replay Advance Your Workflow?
+
+Test Replay complements the existing visual artifacts provided by Cypress Cloud by allowing developers to directly interact with tests as they occurred in CI. It eliminates the need for local test replication by enabling developers to time-travel to the exact moment of test failure, replaying and interacting with network requests, console logs, JavaScript errors, and element rendering. With Test Replay, developers gain access to previously inaccessible details and feedback loops, enabling faster and more accurate debugging, and instilling confidence in handling test failures in CI. Replacing video recordings with Test Replay provides a superior debugging experience.
+
+## Getting Started
+
+:::info
+
+&#8239<Icon name="check-circle" /> Test Replay is available to **all** Cypress Cloud plans at no additional cost, subject to usage limits. Free plans allow up to 500 test results per month, while exceeding this limit requires upgrading the plan.
+
+:::
+
+Test Replay is automatically enabled when updating to `v13` of Cypress and an opt-out is provided through project-level settings. Learn about what this means for capturing video of test failure in Cypress Cloud in the [See also](#See-also) section below.
+
+All you need to do after recording test runs is log in to your Cypress Cloud account and leverage these superior debugging capabilities. No additional software, configuration or changes to the existing test recording process is required. For example, you do **_not_** need to edit your [Cypress configuration](/guides/references/configuration) file or pass any flags or new commands when recording via [the command line](/guides/guides/command-line) in order to enable or use Test Replay.
```

Comment:
superior feels a tad strong here. Maybe "additional" instead?

Reply:
With the video default changes in v13, I think it feels less correct to say "additional", considering the out of the box config. "Improved" sounds underwhelming. Maybe just "new". I'll play around.